### PR TITLE
[WIP] Add Hostel Model

### LINKED ISSUE #119: Hostel field missing in profile route

### PR DESCRIPTION
**Pull request by Saurav-Shrivastav**
This fixes #119

### CODE REVIEW

File: buying_selling/users/models.py [hunk 1]
```diff
@@ -106,6 +114,7 @@ class Profile(models.Model):
         ('BE-MBA(ECE)', 'Electroincs MBA Dual Degree'),
         ('Others', 'Other'),
     )
+    hostel = models.ManyToManyField(Hostel)
```
Comment (jsparmani) [inline buying_selling/users/models.py:117]: Why a `ManyToMany` Field?

Comment (jsparmani) [reply buying_selling/users/models.py:117]: I guess at any point of time we should only keep one hostel linked with a profile

Comment (jsparmani) [reply buying_selling/users/models.py:117]: Everything else is prefect!

Comment (Saurav-Shrivastav) [reply buying_selling/users/models.py]: yes you are right, I made a mistake, it should be foreign key, I've updated it in my new commit

File: buying_selling/users/api/serializers.py [hunk 2]
```diff
@@ -53,6 +54,7 @@ class Meta:
 class ProfileUpdateSerializer(ModelSerializer):
 
     user = UserDetailSerializer()
+    hostel = CharField(source='hostel.name', read_only=True)
```
Comment (jsparmani) [inline buying_selling/users/api/serializers.py:57]: We don't need hostel name in update endpoint.

Comment (Saurav-Shrivastav) [reply buying_selling/users/api/serializers.py]: Yeah, this was a bug. Fixed it!

**Pull request merged**